.ci/setup.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+set -e
+
+packages="
+  c('assertthat', 'covr', 'data.table', 'DT', 'futile.logger', 'ggplot2',
+    'glue', 'igraph', 'knitr', 'magrittr', 'pkgdown', 'R6', 'roxygen2',
+    'rlang', 'rmarkdown', 'testthat', 'visNetwork', 'webshot', 'withr')
+    "
```

Comment:
I've been using this `remotes::install_deps` thing for spongebob that I saw somewhere else, and it seems to work fine. That way we can single-source our dependencies. 

https://github.com/jayqi/spongebob/blob/master/.github/workflows/tests-r-pkg.yml#L41

Reply:
ok